understanding/21/pointer-gestures.html

```diff
@@ -41,7 +41,7 @@ <h2>Intent of this Success Criterion</h2>
 			<p>Examples of path-based gestures include swiping, sliders and carousels dependent on the direction of interaction, and other gestures which trace a prescribed path such as drawing a specific shape. Such paths may be drawn with a finger or stylus on a touchscreen, graphics tablet, or trackpad, or with a mouse, joystick, or similar pointer device.</p>
 
      <h3>The difference between Pointer Gestures and Dragging</h3>
-<p>Dragging is a movement where the user picks up an object with a pointer (such as mouse cursor or a finger) and moves it to some other position. This movement from start point to end point does not require the user to follow any particular path or direction. Dragging is therefore not path-based. In contrast, a path-based pointer gesture requires the traversal of an intermediate point, which is a technical way of expressing that the directionality and possibly speed of the gesture communicates a particular command to the system. Dragging motions are covered in <a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging.html">Success Criterion 2.5.7: Dragging</a>.</p>
+<p>Dragging is a movement where the user picks up an object with a pointer (such as mouse cursor or a finger) and moves it to some other position. This movement from start point to end point does not require the user to follow any particular path or direction. Dragging is therefore not path-based. In contrast, a path-based pointer gesture requires the traversal of an intermediate point, which is a technical way of expressing that the directionality and possibly speed of the gesture communicates a particular command to the system. Dragging motions are covered in <a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging.html">Success Criterion 2.5.7: Dragging Movements</a>.</p>
```

Comment:
```suggestion
<p>Dragging is a movement where the user picks up an object with a pointer (such as mouse cursor or a finger) and moves it to some other position. This movement from start point to end point does not require the user to follow any particular path or direction. Dragging is therefore not path-based. In contrast, a path-based pointer gesture requires the traversal of an intermediate point, which is a technical way of expressing that the directionality and possibly speed of the gesture communicates a particular command to the system. Dragging motions are covered in <a href="dragging-movements">Success Criterion 2.5.7: Dragging Movements</a>.</p>
```

Suggest changing to shortened `href`, and correcting it to `dragging-movements` (the current one still works, but only because the server corrects the URL)